tasks/bump.js

```diff
@@ -10,7 +10,7 @@ module.exports = function (grunt) {
       files: ['package.json'],
       updateConfigs: ['pkg'],
       push: true,
-      pushTo: 'git@github.com:mozilla/fxa-js-client.git master',
+      pushTo: 'git@github.com:mozilla/fxa-js-client.git update-master',
```

Comment:
Not sure if this is the best thing way to do this, but created a separate branch that we can push too and then we can manually merge it to master.